src/config/bootloader.rnc

```diff
@@ -54,19 +54,25 @@ bl_global =
     boot_root? &
     boot_boot? &
     boot_mbr? &
-    stage1_dev?
+    stage1_dev? &
+    boot_efilabel? &
+    prompt? &
+    secure_boot?
   }
 
 lines_cache_id = element lines_cache_id { text }
 stage1_dev = element stage1_dev { text }
 bl_timeout = element timeout { INTEGER }
 gfxmenu = element gfxmenu { text }
+boot_efilabel = element boot_efilabel { text }
 
 # really "true"|"false", without config:type=boolean
 bl_activate = element activate { "true" | "false" }
 boot_root = element boot_root { "true" | "false" }
 boot_boot = element boot_boot { "true" | "false" }
 boot_mbr = element boot_mbr { "true" | "false" }
+prompt = element prompt { "true" | "false" }
+secure_boot = element secure-boot { "true" | "label" }
```

Comment:
Oh, I forgot to submit the latest version (I detected that pitfall when I tried it).